source/includes/_zapier.md

```diff
@@ -0,0 +1,37 @@
+## LOCALiQ App on Zapier®
+
+**Summary:** Zapier® is a tool that allows two systems to exchange data without any coding. LOCALiQ makes available a lead app on the Zapier® platform. You can reach out to [LOCALiQ API Support](mailto:apiservices@localiq.com) to request access.
+
+- The LOCALiQ Zapier® app can be used at no additional cost.
+- Zapier® offers a free trial if you would like to test this solution.
+- The setup can be be completed by client or CSM.
+
+NOTE: There is an additional monthly subscription cost through Zapier® The cost depends on the number of leads sent per month and the number of leads transmitted between the systems (which Zapier® refers to as "Tasks"). Pricing information for Zapier® can be found [here](https://zapier.com/pricing).
+
+**Parameters:**
+
+With the LOCALiQ App you can get contacts and contacts interactions for ingestion into the CRM of your choice. The CRM should also have an app available on Zapier®.
```

Comment:
replace "contacts interactions" with "contact interactions"

Reply:
This has been updated to "contact interactions".